app/views/hyrax/base/_form_progress.html.erb

```diff
@@ -1,3 +1,4 @@
+<%# OVERRIDE: (10-15-17) This file was rebased for Hyrax 2.3.0. Please rebase for future updates. %>
```

Comment:
Wrong year in the date. Also can you use YYYY-MM-DD ?

Reply:
The date and format has been updated.